tests/test_context.py

```diff
@@ -48,3 +49,56 @@ def test_message_handler(device):
     assert context.message_handler is message_handler_test
     with context:
         assert current_context().message_handler is context.message_handler
+
+
+def test_async_wake_interval(device):
+    """Test read-write property async_wake_interval."""
+    with Context(device) as context:
+        context.async_wake_interval = 42
+        assert context.async_wake_interval == 42
+
+
+# THIS DOES NOT WORK, PLEASE HELP!
+
+# def test_is_supported(device):
+#     with Context(device) as context:
+#         assert context == current_context()
+#         with context:
+#             MONO42 = context.is_supported("MONO", "42")
```

Comment:
Are you sure these channel config and sample type are available? I think it's better to check first.

Reply:
It's device dependent, not sure how we can test it.

Reply:
@Huy-Ngo I just randomly pick in `CHANNEL_CONFIGS` and `SAMPLER_TYPES`, no idea on how can we test it

Reply:
Maybe just test for exception handling, e.g. valid and invalid channel conf and sample type, no assertion.  BTW pls keep the PR small (i.e. stop adding stuff to this one), otherwise the review process will never stop.